arangod/Pregel/PregelFeature.cpp

```diff
@@ -95,21 +90,21 @@ std::pair<Result, uint64_t> PregelFeature::startExecution(
   ServerState* ss = ServerState::instance();
 
   // check the access rights to collections
-  ExecContext const* exec = ExecContext::CURRENT;
-  if (exec != nullptr) {
+  ExecContext const& exec = ExecContext::current();
+  if (exec.isSuperuser()) {
```

Comment:
this seems to change the behavior compared to before the change.
Now it will only be true if the current user is the super user, before it was true when authentication was enabled.
Is that intentional?